client/gtk2/ibusimcontext.c

```diff
@@ -1529,6 +1530,69 @@ _set_cursor_location_internal (IBusIMContext *ibusimcontext)
 
     area = ibusimcontext->cursor_area;
 
+#if GTK_CHECK_VERSION (3, 98, 4)
+    graphene_point_t p;
+
+    if (!gtk_widget_compute_point (ibusimcontext->client_window,
+                                   GTK_WIDGET (gtk_widget_get_root (ibusimcontext->client_window)),
+                                   &GRAPHENE_POINT_INIT (area.x, area.y),
+                                   &p))
+        graphene_point_init (&p, area.x, area.y);
+
+    gtk_native_get_surface_transform (gtk_widget_get_native (ibusimcontext->client_window), &nx, &ny);
```

Comment:
It would beyond 80 char columns and I think it's good to create another variable `native = gtk_widget_get_native (ibusimcontext->client_window);`.